### DISCUSSION TITLE
Is there a way to automatically assign issue numbers?

### DISCUSSION THREAD

**Discussion by liverpoolfc-fan** [Z-Hub/Z-Push#11] [Ideas]
In the old days, the Z-Push stash used to automatically assign new issue numbers for tracking changes. Is there a way to do this with github?

**Comment by matidau** [comment]
Github assigns a number for Issues, the same numbering sequence is also shared with PRs and Discussion posts.

For instance this is issue #9

**Reply by liverpoolfc-fan** [reply]
Was wondering if it was possible to configure it to use ZP-nnnn type numbers. From what I have read, it does not seem to be possible.
